webextension/selector/ui.js

```diff
@@ -42,20 +42,13 @@ window.ui = (function () { // eslint-disable-line no-unused-vars
     return false;
   }
 
-  exports.isHeader = isHeader;
-
-  function htmlQuote(s) {
-    s = s + "";
-    return s.replace(/&/g, '&amp;').replace(/"/g, '&quot;').replace(/</g, '&lt;');
-  }
-
   let substitutedCss = inlineSelectionCss.replace(/MOZ_EXTENSION([^\"]+)/g, (match, filename) => {
     return chrome.extension.getURL(filename);
   });
 
   function makeEl(tagName, className) {
     if (! iframe.document) {
-      console.trace();
+      throw new Error("Attempted makeEl before iframe was initialized");
```

Comment:
I find this error a little confusing. If `makeEl` sometimes races with iframe initialization, is there a better way to enforce synchronization, so that this couldn't happen? Maybe a Promise that resolves when the iframe's document or window fires a ready event?